src/views/AgmLocationChg.vue

```diff
@@ -340,6 +371,27 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
     return (this.totalFee > 0)
   }
 
+  /** Array of validations rules for AGM year. */
+  get agmYearRules (): Array<(val) => boolean | string> {
+    const rules = [] as Array<(val) => boolean | string>
+    rules.push(val => !!val || 'Agm year is required.')
+    rules.push(val => (val && +val > 0) || 'Number must be greater than 0')
+    rules.push(val => (val && val.toString().length === 4) || 'Number must be 4 digits')
```

Comment:
You probably don't need to check for > 0 or length != 4. I think those will be implicitly covered by lines 380 and 381 already. Thoughts?

Also, I think AGM needs to be all-caps.